crates/libtiny_tui/src/lib.rs

```diff
@@ -178,9 +178,17 @@ async fn input_handler<S>(
                         if msg[0] == '/' {
                             // Handle TUI commands, send others to downstream
                             let cmd: String = (&msg[1..]).iter().collect();
-                            let handled = tui.borrow_mut().try_handle_cmd(&cmd, &from);
-                            if !handled {
-                                snd_ev.try_send(Event::Cmd { cmd, source: from }).unwrap();
+                            let result = tui.borrow_mut().try_handle_cmd(&cmd, &from);
+                            match result {
+                                CmdResult::Ok => {}
+                                CmdResult::Continue => {
+                                    snd_ev.try_send(Event::Cmd { cmd, source: from }).unwrap()
```

Comment:
When we get `Continue`, pass the command down to the client (cmd.rs) to handle it as well.